MAUI/Scheduler/appointments.md

```diff
@@ -882,6 +882,56 @@ N>
 
 N> [View sample in GitHub](https://github.com/SyncfusionExamples/maui-scheduler-examples/tree/main/RecursiveExceptionAppointment/BusinessObject)
 
+## Suspend and resume the appointment update
```

Comment:
Change this as - Suspend and resume the appointment UI rendering

Reply:
Updated the changes